### PR TITLE
Add send mail function

### PR DESCRIPTION
**Pull request by JjuliaP**
Create example.html
Create send_mail.py
Change eventually settings

## JIRA

* [Main JIRA ticket](http://ssu-jira.softserveinc.com/browse/LVPYTP-7)

## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/eventually/settings.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 """
 
 import os
-
+Temp_Path = os.path.realpath('.')
```
Comment (ssOleg) [inline eventually/eventually/settings.py]: Why does this variable contain camelcase and snake-case convention? `temp_path` or `TempPath`

Comment (JjuliaP) [reply eventually/eventually/settings.py]: This variable is not used. I deleted it

File: eventually/eventually/settings.py [hunk 2]
```diff
@@ -134,3 +133,11 @@
...
 except ImportError:
     pass
+
+#"""This module contains required settings for sending email."""
+EMAIL_USE_TLS = True
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_HOST_USER = 'eventually.proj@gmail.com'
+EMAIL_HOST_PASSWORD = '1234qwerty'
+EMAIL_PORT = 587
+DEFAULT_FROM_EMAIL = 'eventually.proj@gmail.com'
```
Comment (ssOleg) [inline eventually/eventually/settings.py]: `No newline at end of file` :)?

Comment (JjuliaP) [reply eventually/eventually/settings.py]: Now is better)

**Review by ssOleg — APPROVED** [review]

**Pull request merged**